test/python/transpiler/test_sabre_swap.py

```diff
@@ -273,15 +275,21 @@ def test_no_infinite_loop(self, method):
 
         # Assert that the same keys are produced by a simulation - this is a test that the inserted
         # swaps route the qubits correctly.
-        if not optionals.HAS_AER:
-            return
+        with warnings.catch_warnings():
+            # TODO remove Aer stops using Provider Qiskit class
+            warnings.filterwarnings("ignore", category=DeprecationWarning, module="qiskit")
+            if not optionals.HAS_AER:
+                return
```

Comment:
This seems to be caught already by the base class filter, right?
```suggestion
        if not optionals.HAS_AER:
            return
```